product_docs/docs/pem/8/installing_pem_server/pem_server_inst_linux/prerequisites.mdx

```diff
@@ -5,56 +5,71 @@ navTitle: "Prerequisites"
 
-  3.  Verify that the sslutils extension and hstore contrib module are installed on your Postgres server.
+  4.  Verify that the sslutils extension and hstore contrib module are installed on your Postgres server.
 
       -   If you're using EDB Postgres Advanced Server, the sslutils extension and hstore contrib module are installed by default.
 
-      -   If you're using PostgreSQL, make sure you have access to the PostgreSQL community repository, and then install the sslutils extension and the hstore contrib module: 
+      -   If you're using PostgreSQL, make sure you have access to the PostgreSQL community repository ([yum](https://www.postgresql.org/download/linux/redhat/), [apt](https://wiki.postgresql.org/wiki/Apt)), and then install the sslutils extension and the hstore contrib module: 
 
           For RHEL/AlmaLinux/Rocky Linux 8:
           
           ```shell
-          dnf install sslutils-<x> postgresql<x>-contrib
+          dnf install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
           For RHEL/OL/CentOS 7:
           
           ```shell
-          yum install sslutils_<X> postgresql<x>-contrib
+          yum install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
           For Debian/Ubuntu:
           
           ```shell
-          apt install postgresql-<X>-sslutils postgresql<x>-contrib
+          apt install postgresql-<x>-sslutils
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
           For SLES:
           
           ```shell
-          zypper install sslutils_<X> postgresql<x>-contrib
+          zypper install sslutils_<x> postgresql<x>-contrib
           ```
 
-          Where `x` is the Postgres server version.
+          Where `<x>` is the Postgres server version.
 
-      !!! Note
-          Debian 10 and Ubuntu 20 increased the requirements to accept the certificates due to security reason.
+      !!! Note For Debian and Ubuntu users     
+          The postgres community repository for apt (used by Debian and Ubuntu) does not include `sslutils` and therefore this must be installed from the EDB repository along with [PEM](/installing_pem_server_using_edb_repository/index.mdx).
+          
+          Debian 10 and Ubuntu 20 changed the requirements for accepting certificates.
 
-            -   If you want to install the PEM agent on any of the machines, you must upgrade ssltuils to 1.3 where 4096-bit RSA key and sha256 signature algorithm support was added.
+            -   If you want to install the PEM agent on any of the machines, you must upgrade sslutils to 1.3 where 4096-bit RSA key and sha256 signature algorithm support was added.
```

Comment:
I know this isn't new in this PR, but I find the talk of 'upgrading' confusing given that these instructions are for a fresh install and that all the repos (I think) give v1.3 by default. Do we need to mention this at all? If so maybe we can clarify that it only applies if you already have an old version of SSLUtils?

Reply:
updated and committed